### PR TITLE
feat: improve demo notebook with comments

### PR DESCRIPTION
**Pull request by Alikerin**

### CODE REVIEW

File: notebooks/InstaGeo_Demo.ipynb [hunk 1]
```diff
@@ -1,518 +1,886 @@
...
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "8b7a5f61",
+        "outputId": "78a1c750-3a23-4c0a-f45e-94388625fb8b"
+      },
+      "outputs": [],
+      "source": [
+        "# Enter you EarthData user account credentials\n",
+        "USERNAME = getpass('Enter your EarthDAata username: ')\n",
+        "PASSWORD = getpass('Enter your EarthDAata password: ')\n",
```
Comment (rym-oualha) [inline notebooks/InstaGeo_Demo.ipynb:112]: Small typo in "EarthDAata"

File: instageo/apps/viz.py [hunk 2]
```diff
@@ -68,7 +68,6 @@ def add_raster_to_plotly_figure(
     # Reproject to EPSG:3857 CRS
     xarr_dataset = xarr_dataset.rio.write_crs(from_crs).rio.reproject("EPSG:3857")
     xarr_dataset = xarr_dataset.where(xarr_dataset <= 1, 0)
-    xarr_dataset = xarr_dataset.where(xarr_dataset > 0.8, 0)
```
Comment (rym-oualha) [inline instageo/apps/viz.py:71]: Why did you remove this code part ?

File: notebooks/InstaGeo_Demo.ipynb (hunk 1 above)
Comment (Alikerin) [reply notebooks/InstaGeo_Demo.ipynb:112]: Yes, I will fix it.

File: instageo/apps/viz.py (hunk 2 above)
Comment (Alikerin) [reply instageo/apps/viz.py:71]: For locust breeding ground visualisation I was increasing the threshold for class 1 so that we see where the model is very confident. This is not required for the cropland classification data

File: notebooks/InstaGeo_Demo.ipynb (hunk 1 above)
Comment (Alikerin) [reply notebooks/InstaGeo_Demo.ipynb]: fixed

**Review by rym-oualha — APPROVED** [review]
